docs/cookbook.md

```diff
@@ -471,3 +471,9 @@ Extend Temporal to support arbitrarily-large years (e.g., **+635427810-02-02**)
 An example of using `Temporal.TimeZone` for other purposes than a standard time zne.
 
 → [NYSE time zone](cookbook-nyse.md)
+
+### Locked-down Temporal
+
+"Lock down" the Temporal object so that it doesn't leak any information about the host system, and the system clock is controllable, for use in security applications or for mocking in tests.
+
+→ [Locked-down Temporal](cookbook-mock.md)
```

Comment:
Same comment as https://github.com/tc39/proposal-temporal/pull/1370#discussion_r579546707, non-blocking, just curious as it's a departure

Reply:
Answered there, tl;dr: this one is longer and less general-purpose.